src/EnergyPlus/ZonePlenum.hh

```diff
@@ -182,11 +182,11 @@ namespace ZonePlenum {
 
     void UpdateAirZoneSupplyPlenum(EnergyPlusData &state, int const ZonePlenumNum, bool &PlenumInletChanged, bool const FirstCall);
 
-    int GetReturnPlenumIndex(EnergyPlusData &state, int const &ExNodeNum);
+    int GetReturnPlenumIndex(EnergyPlusData &state, int ExNodeNum);
 
-    void GetReturnPlenumName(EnergyPlusData &state, int const &ReturnPlenumIndex, std::string &ReturnPlenumName);
+    void GetReturnPlenumName(EnergyPlusData &state, int ReturnPlenumIndex, std::string &ReturnPlenumName);
```

Comment:
Should really be `std::string GetReturnPlenumName(EnergyPlusData &state, int ReturnPlenumIndex)`

Reply:
Really, all of these objects should inherit from a single NamedObject class and use a single function `std::string GetObjectName(EnergyPlusData &state, Array1D<NameObject> &objects, int index)` rather than this proliferation of component specific lookup functions. I already opened an [issue](https://github.com/NREL/EnergyPlus/issues/8907) about this.